frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.ts

```diff
@@ -29,7 +29,7 @@
 import {Component, ElementRef} from "@angular/core";
 import {I18nService} from "app/modules/common/i18n/i18n.service";
 import {EnterpriseTrialService} from "app/components/enterprise/enterprise-trial.service";
-import {HttpClient, HttpErrorResponse} from "@angular/common/http";
+import {HttpClient} from "@angular/common/http";
 import {NotificationsService} from "core-app/modules/common/notifications/notifications.service";
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/14f0db8dc76db4a9f86d59de6e794b9b8e3702e1/frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>